staging/src/k8s.io/apiserver/pkg/registry/generic/rest/proxy.go

```diff
@@ -214,6 +212,10 @@ func (h *UpgradeAwareProxyHandler) tryUpgrade(w http.ResponseWriter, req *http.R
 	return true
 }
 
+func (h *UpgradeAwareProxyHandler) SendRequest(method string, location *url.URL, header http.Header, body io.Reader) (net.Conn, error) {
+	return h.connectBackend(method, location, header, body)
+}
+
 // connectBackend dials the backend at location and forwards a copy of the client request.
 func (h *UpgradeAwareProxyHandler) connectBackend(method string, location *url.URL, header http.Header, body io.Reader) (conn net.Conn, err error) {
```

Comment:
Yeah, I will do that. I originally had things factored a bit differently, and ended up with the same signature but forgot to do the rename.